src/styles/index.scss

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.center {
+  position: absolute;
+  height: 5vmin;
+  width: 5vmin;
+  background-color: #f6a603;
+  border-radius: 50%;
+  z-index: 10;
+  margin: 0 auto;
+}
```

Comment:
The element 'center' should follow the BEM naming convention. If it's an element of the 'stopwatch' block, it should be named something like 'stopwatch__center'.